Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.10.3
```

Comment:
I think minor version is enough here: `alpine:3.10`

Reply:
Yes, you're right. We can use 3.10 instead
Updated it